core/src/font.rs

```diff
@@ -806,6 +810,9 @@ pub struct Glyph {
 
     shape: GlyphShape,
     advance: Twips,
+
+    // The character this glyph represents.
+    character: char,
```

Comment:
Maybe `Option<char>` instead?

Reply:
If you're referring to the replacement character, I don't believe `Option<char>` is appropriate here, as it seems that Flash just doesn't care whether `u16` is a surrogate code point or not (which is the requirement behind Rust's `char`). However, the replacement character is used everywhere, so I just did the same and added a TODO comment (just now). I believe we need a wider refactor of Flash characters here, but that's outside of this PR.